experimental/generated/csv/fields.csv

```diff
@@ -1263,203 +703,21 @@ ECS_Version,Indexed,Field_Set,Field,Type,Level,Normalization,Example,Description
-8.3.0-dev+exp,true,process,process.parent.group_leader.tty.char_device.minor,long,extended,,128,The TTY character device's minor number.
-8.3.0-dev+exp,true,process,process.parent.group_leader.uptime,long,extended,,1325,Seconds the process has been up.
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.domain,keyword,extended,,,Name of the directory the user is a member of.
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.email,keyword,extended,,,User email address.
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.full_name,keyword,extended,,Albert Einstein,"User's full name, if available."
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.full_name.text,match_only_text,extended,,Albert Einstein,"User's full name, if available."
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.group.domain,keyword,extended,,,Name of the directory the group is a member of.
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.group.id,keyword,extended,,,Unique identifier for the group on the system/platform.
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.group.name,keyword,extended,,,Name of the group.
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.hash,keyword,extended,,,Unique user hash to correlate information for a user in anonymized form.
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.id,keyword,core,,S-1-5-21-202424912787-2692429404-2351956786-1000,Unique identifier of the user.
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.name,keyword,core,,a.einstein,Short name or login of the user.
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.name.text,match_only_text,core,,a.einstein,Short name or login of the user.
-8.3.0-dev+exp,true,process,process.parent.group_leader.user.roles,keyword,extended,array,"[""kibana_admin"", ""reporting_user""]",Array of user roles at the time of the event.
-8.3.0-dev+exp,true,process,process.parent.group_leader.working_directory,keyword,extended,,/home/alice,The working directory of the process.
-8.3.0-dev+exp,true,process,process.parent.group_leader.working_directory.text,match_only_text,extended,,/home/alice,The working directory of the process.
 8.3.0-dev+exp,true,process,process.parent.hash.md5,keyword,extended,,,MD5 hash.
 8.3.0-dev+exp,true,process,process.parent.hash.sha1,keyword,extended,,,SHA1 hash.
 8.3.0-dev+exp,true,process,process.parent.hash.sha256,keyword,extended,,,SHA256 hash.
 8.3.0-dev+exp,true,process,process.parent.hash.sha384,keyword,extended,,,SHA384 hash.
 8.3.0-dev+exp,true,process,process.parent.hash.sha512,keyword,extended,,,SHA512 hash.
 8.3.0-dev+exp,true,process,process.parent.hash.ssdeep,keyword,extended,,,SSDEEP hash.
 8.3.0-dev+exp,true,process,process.parent.hash.tlsh,keyword,extended,,,TLSH hash.
-8.3.0-dev+exp,true,process,process.parent.interactive,boolean,extended,,True,Whether the process is connected to an interactive shell.
```

Comment:
https://github.com/elastic/endpoint-package/blob/49d9a9ac02762afa3ad158febe5edc164883d715/custom_subsets/elastic_endpoint/process/linux_event_model_event.yaml#L82

Reply:
Addressed in https://github.com/elastic/ecs/pull/1847/commits/fec3da59c2089dc1b093eae576fb39018278f977